Code/Editor/Plugins/ComponentEntityEditorPlugin/UI/Outliner/OutlinerWidget.cpp

```diff
@@ -1087,6 +1088,13 @@ void OutlinerWidget::SelectSliceRoot()
 
 void OutlinerWidget::OnEntityPickModeStarted()
 {
+    // If we're in component mode, it's possible the outliner isn't currently enabled. If so,
+    // make sure to enable it while in pick mode.
+    if (!isEnabled())
```

Comment:
suggestion: Might it be worth adding a check (or assert) to check we're in Component Mode at this point and we're not disabled for another reason?